docs/cel2/whats-changed.md

```diff
@@ -0,0 +1,135 @@
+* __epochSnarkData__ - not needed since there will be no [plumo](https://docs.Celo.org/protocol/plumo) support in the Celo L2.
+* __extraData__ - will have the BLS aggregated signature removed, this simplifies the L2 implementation and since we already trust those blocks and distributed rewards for them, the BLS signature is no longer required.
+
+## EIP1559 implementation
+
+Previously our implementation used a smart contract [(here)](https://github.com/celo-org/celo-monorepo/blob/faca88f6a48cc7c8e6104393e49ddf7c2d7d20e3/packages/protocol/contracts-0.8/common/GasPriceMinimum.sol#L162) to calculate the base fee which allowed for governable parameters. Now we use the the standard EIP1559 algorithm with the parameter values being defined in the chain config.
+
+The parameters we are using are:
+
+- Alfajores
+  - EIP-1559 elasticity multiplier: 5
+  - EIP-1559 denominator: 400
+  - EIP-1559 floor: 25 Gwei (Note that the floor is not part of the original EIP-1559 specification, but it did exist in the Celo L1 smart contract implementation)
+
+## RPC API
+
+### Pre-transition data
+
+Old blocks, transactions, receipts and logs will still be accessible via the RPC API but they will differ a bit from the corresponding objects retrieved from the L1 RPC API.
+
+In general the changes involve additional extra unset fields that have been added upstream but were not present on historical Celo L1 objects, and the removal of some unnecessarily set fields on Celo L1 objects.
+
+For in depth details of what is changed see here - (TODO add section in specs covering this)
+
+### Pre-transition execution and state access
+
+RPC API calls for pre-transition blocks that are performing execution or accessing state will not be directly supported by the new Celo L2 implementation, however if this is required you can configure your Celo L2 node to proxy to an archive Celo L1 node for these calls. See [here](./l2-operator-guide#supporting-historical-execution)
+
+
+# Optimism → Celo L2
+
+## Blocks
+
+Celo L2 block times are 1s as opposed to 2s for optimism, the gas limit per block remains the same.
+
+## Native token
+The native token is Celo as opposed to Eth. The native token is also an ERC20 token.
+
+## New transaction type
+
+Type 123 (`0x7b`) transaction type allows paying for gas in currencies other than the native asset (Celo). It has an additional field `feeCurrency` which allows the sender to chose the currency they pay gas in a chosen fee currency.
+
+The fee currencies available at mainnet launch will be:
+
+ - USDC (USDC)
+ - Tether USD (USD₮)
+ - PUSO (PUSO)
+ - ECO CFA (eXOF)
+ - Celo Kenyan Shilling (cKES)
+ - Celo Dollar (cUSD)
+ - Celo Euro (cEUR)
+ - Celo Brazilian Real (cREAL)
+
+## L1 fees
+
+In the optimism model, an extra fee is added on to user transactions in order to cover the cost of the L1, this can be surprising to users since they have no visibility about what that fee may be since it is not included in the results of calling estimate_gas.
```

Comment:
```suggestion
In the Optimism model, an extra fee is added on to user transactions in order to cover the cost of the L1, this can be surprising to users since they have no visibility about what that fee may be since it is not included in the results of calling `estimate_gas`.
```